Website/src/rendering/src/assets/css/components/PageContent/PreviewSearch.css

```diff
@@ -0,0 +1,100 @@
+.preview-search {
+  @apply absolute w-screen top-8 left-0;
+  @apply lg:top-14;
+
+  @media (min-width: 1024px) {
+    left: -148px;
+    padding-left: 20px;
+    padding-right: 20px;
+  }
+
+  .preview-search-modal-container {
```

Comment:
You are right, it is good for the lg breakpoint but for widths larger than this the search input size/ position above changes and that makes also the modal position underneath to change. I didn't add breakpoints for this because of the non-support in Horizon but will update with xl, 2xl.